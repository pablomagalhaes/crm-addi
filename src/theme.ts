import { DefaultTheme } from 'styled-components';

const theme: DefaultTheme = {
  color: {
    white: '#FFF',
    black: '#424242',

    background: '#f2f2f2',
    border: 'rgba(0,0,0,0.1)',

    primary: '#6272a4',
    secondary: '#44475a',

    success: '#2AD178',
    warning: '#FABD59',
    danger: '#FF5B68',
  },

  font: {
    family: {
      primary: "'Roboto', sans-serif",
    },

    weight: {
      regular: 400,
      semiBold: 600,
    },

    color: {
      primary: '#424242',
      secondary: '#969696',
    },
  },

  borderRadius: {
    default: '3px',
  },

  breakpoint: {
    sm: 576,
    md: 768,
    lg: 992,
    xl: 1300,
  },
};

export default theme;
