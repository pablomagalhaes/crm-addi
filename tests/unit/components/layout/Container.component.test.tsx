import React from 'react';
import { shallow } from 'enzyme';

import Container from '@/components/layout/Container.component';
import theme from '@/theme';

describe('<Container />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<Container theme={theme} />);
    expect(wrapper).toMatchSnapshot();
  });
});
