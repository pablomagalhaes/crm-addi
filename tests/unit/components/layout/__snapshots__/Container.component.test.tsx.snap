// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should match snapchat 1`] = `
<div
  className="sc-bdnylx cNGQIZ"
/>
`;
